interview-questions.md

```diff
@@ -8,36 +8,97 @@ Then, research the question to expand on your answer. Even if you feel you have
+
+rails generate model -Ruby class that is used to represent data, models can interact with the database through  Rails Active Record. 
+
+rails generate migration -is a way to make changes to your database. you can use key words such as , add_column, remove_column , add_reference, create_table, change_table,change_column, add_belongs_to and add_foreign_key. to use your rails magic on whatever column.
+
+
 4. Consider the Rails routes below. What is the name of the controller method that would be called by each route? What action would each of the controller methods perform?
 
-action: "GET" location: /students
+action: "GET" location: /students -show-all of the instances in students
+
+action: "POST" location: /students -create-a new student instance
+
+action: "GET" location: /students/new -new- the form that you fill out to create a new instance
 
-action: "POST" location: /students
+action: "GET" location: /students/2- show - id- 2/ student 
 
-action: "GET" location: /students/new
+action: "GET" location: /students/2/edit -the form that you fill out to make changes to an existing instance
 
-action: "GET" location: /students/2
+action: "PATCH" location: /students/2 -updates - id- will make that changes made in the edit-verb
 
-action: "GET" location: /students/2/edit
+action: "DELETE" location: /students/2 - destroy - id:2 
 
-action: "PATCH" location: /students/2
 
-action: "DELETE" location: /students/2
 
 5. As a developer, you are making an application to manage your to do list. Create 10 user stories that will help you get your application started. Read more about [user stories](https://www.atlassian.com/agile/project-management/user-stories).
+
+
+Epic- make an application to manage your to do list
+
+
+
+
+Create 10 user stories that will help you get your application started
+
+
+
+
+Stories fragments
+
+
+-As a user i can see a page that ask me to sign in or create an account. 
+
+-As a user i can create an account , the user name has to be unquie and the password cant be pass word or less than 7 chars.
+
+-As a user i can see a words on the screan that says "${USERS NAME}" Blogs and has a button that can create more blogs 
+
+-As a user i can click the create new Blog button and be directed to the creaion page
+
+-As a user i can see a screen that has a form that i can fill out that say blog title and blog content.
+
+-As a user i can fill out the form and am rediected to the main screen and i see my newly created blog entry
+
+-As a user i can edit my my current blogs 
+
+-As a user i can see a form that i can make updates to my current blog sorted by id:
+
+-As a user i can submit that update and am redirected to the main screen and i see my updated blogs
+
+-As a user i can Delete a blog andwhen i click that button i see a model that ask me if im sure i want to delete.
+
+-As a user i  am redirect back to the main screen and i see that my blog instance has been deleted.
```

Comment:
Nice job with the user stories! Love that you are thinking about redirects and modals to confirm deletion.